app/main.py

```diff
@@ -2,4 +2,78 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list = None
```

Comment:
```suggestion
            coords: Optional[list[int]] = None
```
It is optional because it accepts None as default